CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.6)
+
+if (DEFINED ENV{VCPKG_ROOT})
+  set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake")
+endif ()
+
+set(n clinfo)
+project(${n})
+
+find_package(OpenCL REQUIRED)
```

Comment:
Another sensible requirement would be CMAKE_C_STANDARD, 99 works.

Reply:
I suggest we keep this file simple for now. I hope for the repo owner to recognise how minor this addition is.

Reply:
That's fine by me